redbot/core/core_commands.py

```diff
@@ -3633,7 +3633,19 @@ async def _set_api(
         else:
             if ctx.bot_permissions.manage_messages:
                 await ctx.message.delete()
+            embed = discord.Embed()
+            for dict_key, token in tokens.items():
+                if token.startswith("<") and token.endswith(">"):
+                    angle_bracket_warning = _(
+                        "Your `{dict_key}` was set including the <> symbols. If this was an accident you may experience"
```

Comment:
For translatable strings, I would personally suggest naming the formatting variable something contextual so that translators are better aware of the translation context for example "api_service_name"